dist/images/ovnkube.sh

```diff
@@ -524,12 +524,27 @@ ovn-northd () {
   fi
 }
 
+# make sure ports 6641/6642 are open on master
+iptables-rules () {
+iptables -C INPUT -p tcp -m tcp --dport 6642 -m conntrack --ctstate NEW -j ACCEPT
+if [[ $? != 0 ]] ; then
+  iptables -A INPUT -p tcp -m tcp --dport 6642 -m conntrack --ctstate NEW -j ACCEPT
```

Comment:
We probably want -I here (not -A) to ensure our rules are first? Otherwise they might end up after default rules which sometimes reject traffic explicitly. Eg default Fedora firewall rules end with:

-A INPUT -j REJECT --reject-with icmp-host-prohibited

and if we append ours, then it'll end up after this last rule and won't work.

Reply:
@dcbw how are you going to avoid someone who comes along later and does this
`iptalbes -I INPUT -j REJECT --reject with icmp-host-prohibited`?

Reply:
@girishmg you can't, and that's the problem with shared iptables :) But those kinds of reject-all rules are almost always default rules added at startup, so we can almost always assume we get run later and supercede those.

But we also often have resync loops in other projects (kube-proxy, kubernetes, openshift-sdn, etc) that refresh the iptables rules to avoid ordering issues.

This isn't a new problem though, it's always existed...

Reply:
@dcbw Oops, ment -I - good catch.